### ISSUE TITLE
Update Deno Deploy github app description?

### ISSUE THREAD

**Issue by jedahan** [denoland/deploy_examples#15]
Went to sign up for Deno Deploy, and the permissions it asked for included the all-encompassing`Act on your behalf` permissions.

I clicked [Learn more about Deno Deploy](https://github.com/apps/deno-deploy) which did not give any additional information.

Is this repository the correct place to request explaining (and hopefully reducing) the permissions needed?

**Comment by lucacasonato** [comment]
`Act on your behalf` is not an all-encompassing permission. It just means that if we asked for any permissions which allowed us to act as your user account (which we do not, we only request readonly email access), we would be able to act as your account. This is the minimal set of permissions you can request from GitHub as a GitHub App.

More info: https://github.community/t/why-does-this-forum-need-permission-to-act-on-my-behalf/120453

A description has now been added to the GitHub App.

---

TLDR:
- We can only read-only access your email (read-only), and status checks and source code for repos your explicitly add the app to.

**Comment by jedahan** [comment]
Thanks for the pointer to that thread, though it kind of raised more questions than it answered.

So you are saying even though the wording is `Act on your behalf`, trust the app will only request
* read-only email *address* access (presumably to send email alerts?)
* status checks (what is a status check? like, reading the state of PRs?)
* the ability to ask to read a repositories source code

**Comment by jedahan** [comment]
I guess if you only request read-only email access, its a bit confusing why it should be a github app at all. Especially after skimming https://docs.github.com/en/developers/apps/about-apps where there is an explicit callout:

> Don't use a GitHub App if you just need a "Login with GitHub" service. But a GitHub App can use a user identification flow to log users in and do other things.

**Comment by lucacasonato** [comment]
We also allow you to install the app on repositories. That is why it is a GitHub App. The 'can act on your behalf' wording is rather dubious - not sure what GitHub means with that. We specifically only request read only email access during signin. It is not possible for us to do things like post comments, or push to repos as your user account.

**Comment by lucacasonato** [comment]
I'll close this as unactionable then.
